tools/monitoring/config.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+)
+
+var databaseConfig MySQLConfig
```

Comment:
can we extend it instead? 
```
type DBConfig struct {
MYSQLConfig
}
```

Reply:
MySQLConfig is defined in db_mysql.go. It is not the default mysql config. Renamed to DBConfig so it's clear.